pallets/runtime-upgrade/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
```

Comment:
Question: Runtime upgrade is done by uploading compressed wasm of runtime, right? Then why this pallet?

Reply:
This pallet sets the upgrade origin.

Reply:
Upgrade origin is by default the Account used to run runtime upgrade, right ? I did not understand this. Can you plz go little deeper?